lib/linter.coffee

```diff
@@ -45,7 +47,9 @@ class Linter
   findHamlLintYmlFile: (filePath) =>
     new Promise (resolve, reject) =>
       @findFile filePath, '.haml-lint.yml'
-      .then (hamlLintYmlPath) ->
+      .then (hamlLintYmlPath) =>
+        if hamlLintYmlPath is undefined
+          hamlLintYmlPath = @findFile filePath, @globalHamlLintFile
```

Comment:
This is just duplicating the current behavior, but with a custom name. I think you meant `@exists @globalHamlLintFile` here?